### PR TITLE
[TASK] Remove RenameClassNonPhpRector from the defaults

### PR DESCRIPTION
**Pull request by sabbelasichon**
Resolves: #6361

### CODE REVIEW

**Comment by sabbelasichon** [comment]
@TomasVotruba We have to be aware that this is breaking form some people. So i added it to the template file to show people that this has changed.

File: templates/rector.php.dist [hunk 1]
```diff
@@ -19,6 +19,9 @@ return static function (ContainerConfigurator $containerConfigurator): void {
     // get services (needed for register a single rule)
     // $services = $containerConfigurator->services();
 
+    // Non PHP Changes - To keep the old behaviour =< 0.10.5 add this
+    // $services->set(RenameClassNonPhpRector::class);
```
Comment (TomasVotruba) [inline templates/rector.php.dist:23]: This can be removed, as template is always generated with the latest version.

**Comment by TomasVotruba** [comment]
Btw, what set configs this rule should be included in?

**Comment by sabbelasichon** [comment]
I don´t know. Never used this. Where does it come from?

**Comment by TomasVotruba** [comment]
I think from `RenameClassRector` where the class should be renamed in framework specific non-PHP files:

- Laravel → `.blade.php`
- Nette → `.neon`
- Symfony → `.xml`, `.yaml`

**Comment by sabbelasichon** [comment]
If you like we could add it to the sets of these framework packages and we should be fine. 
Btw. the whole documentation part goes here: #6374

**Comment by TomasVotruba** [comment]
> add it to the sets of these framework packages and we should be fine

That would be great :+1:  :bow: 

The 3 above should be fine.

**Pull request merged**